app/models/dataset.rb

```diff
@@ -42,8 +42,8 @@ def is_readonly?
     end
   end
 
-  def datafiles
-    links + docs
+  def links
```

Comment:
would it possible to have `has_many :links` instead of this?

Reply:
Adding to next PR